APolitcalApp/src/views/ViewProfilePage.vue

```diff
@@ -13,8 +36,23 @@
       <ToolbarProfilePersonal></ToolbarProfilePersonal>
     </template>
     <template #ContentSlot>
-      <h1>profile</h1>
-      <AuthAccount @sign-out="authStore.logout()" v-if="authStore.session" :session="authStore.session" />
+      <div class="flex flex-col gap-5">
+        <AuthAccount @sign-out="authStore.logout()" v-if="authStore.session" :session="authStore.session" />
+        <div class="flex w-full justify-between">
+          <div>
+            <p>Your Petitions</p>
+            <div class="flex flex-col gap-2">
+              <CardPetitionTitle v-for="petition in petitions" :key="petition.id" :petition-id="petition.id" :petition-title="petition.title" />
```

Comment:
When clicking on a petition it redirects to url.com/review/petitionId, but the review page is only setup to handle url.com/review?id=petitionId

Reply:
I reused this component but I will look